robot_sort/robot_sort.py

```diff
@@ -97,8 +97,23 @@ def sort(self):
         Sort the robot's list.
         """
         # Fill this out
-        pass
-
```

Comment:
All tests passing Bryce, well done building out your `sort()` with the existing helpers 👍